src/Image.svelte

```diff
@@ -1,10 +1,23 @@
 <script>
-  /**
-   * Cloudinary Image Component
-   */
+    /**
+     * Cloudinary Image Component
+     */
+    import {onMount, afterUpdate} from 'svelte';
+    import {getImageTag, responsive} from "./utils";
```

Comment:
Can we rename `responsive` to something more explanatory? like `makeTagResponsive`?

Reply:
renamed to makeElementResponsive